siem_integrations/clx_query/default/clx_query_setup.conf

```diff
@@ -0,0 +1,3 @@
+[setupentity]
+clx_hostname = localhost
+clx_port = 8998
```

Comment:
Is the port easy to configure? Is this the only place where it needs to be changed?

Reply:
this is the only place it needs change